### PR TITLE
Add design pod

### PR DESCRIPTION
**Pull request by AugustinGrigorov**
Added design team and updated the current people.

### CODE REVIEW

**Comment by KwanMan** [comment]
@AugustinGrigorov will this break if someone has the pod set as "growthOne"/"growthTwo" ?

**Comment by AugustinGrigorov** [comment]
I mean, yes, but now I'm thinking that going into the extension settings to switch is not a good approach too, I should probably add a dropdown, right?

**Comment by KwanMan** [comment]
I agree, but let's keep the PRs clean and fall to scope creep.

For now, can you add a check to make sure the pod that is stored is still supported (and ignore it if it's not or some other nice way to handle it). And then feel free to optimise with a dropdown later in another PR

**Comment by AugustinGrigorov** [comment]
will this do?

File: src/popup.js [hunk 1]
```diff
@@ -22,17 +22,21 @@ var kickoff = async function (force) {
   } else if (!pod) {
```
Comment (KwanMan) [inline src/popup.js]: How about
`else if (!pod || !csesByPod[pod]) {
  window.chrome.runtime.openOptionsPage()
}`

Comment (AugustinGrigorov) [reply src/popup.js]: Yeah that makes sense

**Comment by AugustinGrigorov** [comment]
So all good now?

**Comment by KwanMan** [comment]
will release after lunch

**Pull request merged**